### PR TITLE
Temporary fix for mapping components to use  new session data structure

### PR DESCRIPTION
**Pull request by fua0004673**
@shankari  this is a temporary fix to get something at least working before you guys deploy tomorrow. This does however at this moment employ the use of fake data (via turf), however still allows us to generate the visualization below:

Screen Shot 2020-07-27 at 7 32 36 PM

A few notes:
1. As per my previous email and [this](https://github.com/NREL/crowdsource-mask-wearing/issues/21#issuecomment-664730897), currently I am not able to find a generic `GET` for `maskreport/session/{id}` that returns a generic session by ID (which is required with current architecture to obtain the location of a maskreport). Ideally, we don't need to make this API call at all and the location is part of the response object for maskreports. 
2. The generated data will only appear in a small bounding box in the downtown Seattle region (this bbox is hardcoded). I'm using turf's `randomPoint` method
3. Everything else (selection, data popup) still works, but operates off fake data. Changing away from fake data should be relatively straighforward (see map-data.service.ts)
4. Note that currently, the map page shows all data (and doesn't have an option to toggle to only display current user data). The `map-data.service.ts` has been setup to include references to only the user data, but I have yet to develop a UI for toggling between the two modes (going to defer to @shankari for feedback on this)

### CODE REVIEW

**Comment by fua0004673** [comment]
As per [Shankari’s comment](https://github.com/NREL/crowdsource-mask-wearing/issues/21#issuecomment-664772687), will now use sessions directly instead of mask report as a data input source

**Comment by shankari** [comment]
Session get calls https://github.com/NREL/crowdsource-mask-wearing/pull/27

**Comment by fua0004673** [comment]
@shankari, these changes should be good now to integrate the sessions endpoints. Note that since we no longer have fake data the viz is relatively uninteresting

File: frontend/src/environments/environment.ts [hunk 1]
```diff
@@ -4,7 +4,7 @@
 
 export const environment = {
   production: false,
-  apiEndpoint: "http://localhost:3000/api",
+  apiEndpoint: "http://localhost:8080/api",
```
Comment (shankari) [inline frontend/src/environments/environment.ts:7]: Can we please not check this in? Can you just put it into `.gitignore`?

Comment (fua0004673) [reply frontend/src/environments/environment.ts:7]: Ah yes, apologies, will add the environment files to our gitignore 😄

**Pull request merged**